Source/AliveLibAO/CheatController.cpp

```diff
@@ -63,12 +63,12 @@ const InputCommands sCheatKeyArray_FartGas_4C50B8[] =
 
 const InputCommands sCheatKeyArray_VoiceLocks_4C50C8[] =
 {
-    eFartOrRoll,
+    eCrouchOrRoll,
     eDown,
     eThrowItem,
     eDoAction,
     eDown,
-    eFartOrRoll,
+    eCrouchOrRoll,
```

Comment:
isn't this here twice?

Reply:
this is a cheat button sequence, I think multiple entries are fine here (eDown is also here twice)